Makefile

```diff
@@ -432,6 +443,21 @@ htscodecs/htscodecs:
 	  official releases from https://www.htslib.org/\\n" ; \
 	fi
 	@false
+
+# Build the htscodecs/htscodecs/version.h file if necessary
+htscodecs/htscodecs/version.h: force
+	@if test -e htscodecs/.git && test -e htscodecs/configure.ac ; then \
```

Comment:
I see no else clause here.  I tried hiding my .git directory and sure enough it fails.

We need this to be working for the tarball.

Reply:
The idea is that non-git distributions (i.e. the tarball) will supply a version of this file that has the correct version number in it.  If it isn't there then the distribution is incomplete.

Reply:
Ah fair enough and makes sense.  I'll delay my assessment then until the tarballs are ready.